core/src/main/java/io/bisq/core/dao/compensation/CompensationRequest.java

```diff
@@ -108,4 +122,17 @@ public static CompensationRequest fromProto(PB.CompensationRequest proto) {
                 proto.getWaitingForVotingPeriod(),
                 CollectionUtils.isEmpty(proto.getExtraDataMap()) ? null : proto.getExtraDataMap());
     }
+
+    /// API
+    public Coin getRequestedBsq() {
+        checkNotNull(payload);
+        return payload.getRequestedBsq();
+    }
+
+    public Address getIssuanceAddress (BsqWalletService bsqWalletService) {
+        checkNotNull(payload);
+        // Remove leading 'B'
+        String underlying_btc_address = payload.getBsqAddress().substring(1, payload.getBsqAddress().length());
```

Comment:
We should stick with camelCase style: underlying_btc_address -> underlyingBtcAddress

Reply:
Thanks for fixing, it's sometimes hard to kill those habits.